### PR TITLE
Use addScale in Pow operation for tensor+scalar op

### PR DESCRIPTION
**Pull request by qiubit**
This should fix unhandled case where we try to raise multidimensional tensor to scalar exponent (currently this throws assertion).

### CODE REVIEW

File: builtin_op_importers.cpp [hunk 1]
```diff
@@ -1143,8 +1143,61 @@ DEFINE_BUILTIN_OP_IMPORTER(Pad) {
...
+    // TODO: Check "broadcast" and "axis" attributes
+    return combineTensorsElementwise(
+        ctx, inputs, nvinfer1::ElementWiseOperation::kPOW);
+  } else if( inputs.at(0).is_weights() && inputs.at(1).is_weights() ) {
+    // TODO: Add weights together (i.e., constant fold)
+    return MAKE_ERROR("Both inputs are weights",
+                      ErrorCode::kUNSUPPORTED_NODE);
+  } else {
+    // We have two inputs, one is tensor, another are weights
+    auto* tensor_ptr = (inputs.at(0).is_tensor() ?
```
Comment (goldgeisser) [inline builtin_op_importers.cpp]: Why use auto* rather than auto

Comment (goldgeisser) [inline builtin_op_importers.cpp]: The implementation is similar to the implementation of DEFINE_BUILTIN_OP_IMPORTER(Add) , same file line 304

We eventually should factor the common code into a separate function

**Comment by benbarsdell** [comment]
@yinghai after #47 is merged, I have another PR that will supersede this one.

**Comment by benbarsdell** [comment]
Closing this since it's been superceded by #53

**Pull request closed without merging**